modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -80,13 +80,19 @@ This option requires an Enterprise license.
 This mode is not recommended for production clusters. Only set to `off` if you need to move partitions manually.
 |===
 
-== Partition balancing with Kafka API
+== Manually move partitions
 
-As an alternative to Redpanda partition balancing, you can change partition assignments explicitly with the Kafka API or with any 3rd-party tool in the Kafka ecosystem that controls partition movement using the Kafka API.
+As an alternative to Redpanda partition balancing, you can change partition assignments explicitly with `rpk cluster partitions move`.
 
-To reassign partitions with the Kafka API:
```

Comment:
@micheleRP Im not sure we should completely eliminate the Kafka API bit here, because some customers have asked for that by name, and if you're using Kafka ecosystem tools to automate this they're using the API nor RPK. 

 We may want to just leave a note at the bottom says "If you prefer, Redpanda also supports the use of the AlterPartitionAssignments Kafka API and using standard kafka tools such as  kafka-reassign-partitions.sh.

Or we may just want to keep this whole API section and put it at the bottom as its kind of the last resort option (or if your Kafka tooling depends on it)

Reply:
agreed

Reply:
Ok, thanks. I will add a note at the bottom of the section.